app/jobs/exchange_uphold_code_for_access_token_job.rb

```diff
@@ -12,6 +12,7 @@ def perform(publisher_id:)
       # The code acquired from https://uphold.com/authorize is only good for one request and times out in 5 minutes
       # it should now be cleared
       publisher.uphold_code = nil
+      publisher.uphold_updated_at = Time.now
```

Comment:
Minor: We've repeated this line -- to update the uphold_updated_at -- several times so I'd consider an Publisher model callback to update it automatically e.g.:
`before_validation :update_uphold_updated_at, if: -> { exists? && uphold_code_changed? }`